app/controllers/socure_webhook_controller.rb

```diff
@@ -7,14 +7,26 @@ class SocureWebhookController < ApplicationController
   check_or_render_not_found -> { IdentityConfig.store.socure_enabled }
   before_action :check_token
   before_action :check_socure_event
+  # before_action :check_doc_capture_session
 
   def create
     log_webhook_receipt
+    fetch_results if socure_params[:event][:eventType] == 'DOCUMENTS_UPLOADED'
     render json: { message: 'Secret token is valid.' }
   end
 
   private
 
+  def fetch_results
+    docv_transaction_token = socure_params[:event][:docVTransactionToken]
+    dcs = DocumentCaptureSession.find_by(socure_docv_transaction_token: docv_transaction_token)
+    SocureDocvResultsJob.perform_later(
```

Comment:
what do you think about not adding this job to the queue if document_capture_session does not exist? 🤔 and instead notifying NewRelic here

Reply:
done in 3ae82fd635